prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -704,6 +704,13 @@ class WorkflowEngine(
         }
     }
 
+    /**
+     * Fetch the ItemLineages that match the parent of a given [report]
+     */
+    fun fetchItemLineagesForReport(report: Report): List<ItemLineage>? {
```

Comment:
I'm not sure that you need to add this here? 

I think instead you can do `BaseEngine.databaseAccessSingleton.fetchItemLineagesForReport` where this is used.

Reply:
That worked! Updated...